python/grass/pygrass/raster/rowio.py

```diff
@@ -18,17 +18,17 @@
 )
 
 
-def getmaprow_CELL(fd, buf, row, l):
+def getmaprow_CELL(fd, buf, row):
```

Comment:
Does this affect compatibility with the C-libraries or its Python wrappers (like ctypes)
Who can answer this?

Reply:
That's the only point that needs to be answered by someone else

Reply:
If you aren't able to get feedback and really want to get it finished, I'm comfortable with the remaining changes, if you take the changes of the function signature out for another PR.

Reply:
@nilason Do you have an idea on the impact of this, as it seems to be related to the python-c library interfacing.

Reply:
[Rast_get_c_row()](https://grass.osgeo.org/programming8/raster_2get__row_8c.html#a7879236341a1d0ad962ba82e82576093) takes three parameters, and `l` was never used so this shouldn't cause any trouble. It is probably a leftover from earlier implementations.

Reply:
The unused `l` was there from the start https://github.com/OSGeo/grass/commit/0982f55fa6b3b31e4577bacfdab7d2bbea8c18d3

Reply:
FWIW, a historical pointer here is 

"List of functions moved from GIS to Raster library"
https://trac.osgeo.org/grass/wiki/Grass7/RasterLib/ListOfFunctions

Reply:
Little update here, it seems that ctypes has a problem that seems to be related to this, on windows:

![IMG_7696](https://github.com/user-attachments/assets/430dd1da-1a9e-497f-888a-8a05b70878f5)
![IMG_7697](https://github.com/user-attachments/assets/7c704877-9333-40ba-a6ce-4bf45a115c34)
 
 
It’s from a run on main from today: https://github.com/OSGeo/grass/actions/runs/10012925978/job/27679432818#step:11:1495